en/contributing/build.md

```diff
@@ -186,19 +186,19 @@ make android install
 
 ## Build for iOS {#build_cpp_iOS}
 
-> **Warning** You must first build the C++ Library (as shown above).
+> **Warning** You must first build the C++ Library with `mavsdk_server` (i.e. with `-DBACKEND=ON`, as shown [Building the Backend](#build_backend)).
```

Comment:
Just to confirm, to build for iOS you first build the backend (covered in that section below), then you build CPP library for iOS - that is what is covered here right? 
Then there is a further step not covered here for building swift or some other library to run witht he SDK - right?

1. It might be worth spelling this all out here rather than just forward linking to building the backend. Minimally you should explain why you must first build the server.
2. There is an inconsistency in terminology now - you refer to both mavsdk_server and the backend. One or the other. If it were me I would change the name of the section Building the Backend to Building the MAVSDK Server - you can choose to leave the link anchor the same if you want. 

I would also add a paragraph in that section explaining what the server does for you (ie after "The MAVSDK programming-language-specific libraries (e.g. Swift, Python) share a common backend, which may optionally be built as part of the C++ library.")

Reply:
Got it! Will do! :blush: